mmv1/third_party/terraform/services/resourcemanager/ephemeral_google_service_account_jwt.go

```diff
@@ -51,7 +52,7 @@ func (p *googleEphemeralServiceAccountJwt) Schema(ctx context.Context, req ephem
 				Optional:    true,
 				Description: "Number of seconds until the JWT expires. If set and non-zero an `exp` claim will be added to the payload derived from the current timestamp plus expires_in seconds.",
 				Validators: []validator.Int64{
-					fwvalidators.PositiveIntegerValidator(),
+					int64validator.AtLeast(0),
```

Comment:
```suggestion
					int64validator.AtLeast(1), // Must be greater than 0
```

Reply:
was just making the commit about this. You beat me! 😆